src/app-logic.js

```diff
@@ -58,30 +33,31 @@ export default function useAppLogic() {
 
         totalActiveTokens = totalActiveTokens.plus(totalActive)
 
-        const myStake = proposal.stakes.find(
-          stake => stake.entity === connectedAccount
+        const myStake = proposal.stakes.find(stake =>
+          addressesEqual(stake.entity, account)
         )
 
         if (myStake) {
           myStakes.push({
-            proposal: proposal.id,
+            proposalId: proposal.id,
             proposalName: proposal.name,
-            stakedAmount: myStake.amount,
+            amount: myStake.amount,
           })
         }
         return { myStakes, totalActiveTokens }
       },
       { myStakes: [], totalActiveTokens: new BigNumber('0') }
     )
-  }, [proposals, connectedAccount, stakeToken])
+  }, [account, proposals, stakeToken])
+
+  const actions = useProposalActions(proposalPanel.requestClose)
 
   return {
+    actions,
+    isLoading: isLoading, // TODO: Add loading flag for block back again
```

Comment:
i think that now we are able to add that flag again right?

Reply:
Not yet, the blocks start to come when the account is enabled since it using the `ethers` from  `useWallet`. We need to resolve this by using our own created provider. Will work on it on a separate PR

Reply:
right!